js/script.js

```diff
@@ -0,0 +1,141 @@
+const startBtn = document.getElementById('start')
+const gameBoard = document.getElementById('game-board')
+const completionPage = document.getElementById('completion-page')
+const attemptsDisplay = document.getElementById('attempts')
+const playAgainBtn = document.getElementById('play-again')
+const timerDisplay = document.querySelector('.stop-watch')
+
+let cards = []
+let flippedCards = []
+let matchedCard = []
+let attempts = 0
+
+// start button event
+startBtn.addEventListener('click', () => {
+  startTimer()
+  startBtn.classList.add('hidden')
+  gameBoard.classList.remove('hidden')
+})
+
+// Card data and douplicate the pairs
+const allCards = ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H']
+// const allCards = [...cardValues, ...cardValues]
+
+// Shuffle Cards
+function shuffle (array) {
+  for (let i = array.length - 1; i > 0; i--) {
+    const j = Math.floor(Math.random() * (i + 1))
+    console.log(`Swapping elements at indexes ${i} and ${j}`)
```

Comment:
remove from your code once you are done. 

console.log is for debugging your work while developing